src/index.html

```diff
@@ -3,11 +3,287 @@
+        </a>
+
+        <div class="navigation__buttons">
+          <a
+            class="close"
+            href="#home"
+          >
+            <img
+              class="close__icon"
+              src="images/Icon-Close.svg"
+              alt="close">
+          </a>
+        </div>
+      </navigation>
+
+      <navigation class="links menu__links">
+        <ul class="links__list">
+          <li class="links__item"><a class="links__link" href="#products">Products</a></li>
+          <li class="links__item"><a class="links__link" href="#store">Store</a></li>
+          <li class="links__item"><a class="links__link" href="#">Login</a></li>
+          <li class="links__item"><a class="links__link" href="#">Cart(0)</a></li>
+        </ul>
+      </navigation>
+
+      <p class="menu__paragraph">+1 234 5555-55-55</p>
+      <p class="menu__paragraph">call to order</p>
+    </div>
+
+    <main class="main" id="products">
+      <h2 class="title title__secondiary main__title">
+        Recommended
+      </h2>
+
+      <section class="section main__section main__section--margin">
+        <article class="section__article section__article--main">
+          <img
+            class="section__picture"
+            src="images/section1.jpg"
+            alt="phone1"
+          >
+          <h3 class="section__title">
+            Phone (1)
+          </h3>
+          <p class="section__paragraph section__paragraph--main">
+            Designed with instinct, to bring joy back to the everyday. Through the Glyph Interface, a perfected OS and
+            exceptional
+            dual camera. All startlingly fast.
+          </p>
+          <p class="section__price">
+            $ 499.00
+          </p>
+        </article>
+        <article class="section__article section__article--main">
+          <img
+            class="section__picture"
+            src="images/section2.jpg"
+            alt="phone2">
+          <h3 class="section__title">
+            Ear (2)
+          </h3>
+          <p class="section__paragraph section__paragraph--main">
+            Perfecting the path from artist to ear. So that music sounds just as it was intended.
+          </p>
+          <p class="section__price">
+            $ 149.00
+          </p>
+        </article>
+        <article class="section__article section__article--main">
+          <img
+            class="section__picture"
+            src="images/section3.jpg"
+            alt="phone2">
+          <h3 class="section__title">
+            Ear (stick)
+          </h3>
+          <p class="section__paragraph section__paragraph--main">
+            Tech you can &#x2019; feel. Leaving just space for sound to come to life. Powered by custom sound technology.
+          </p>
+          <p class="section__price">
+            $ 119.00
+          </p>
+        </article>
+      </section>
+
+      <h2 class="title title__secondiary title__secondiary--browse main__title" id="store">
```

Comment:
'title__secondiary' used as a class name seems to be misspelled. Correct spelling should be 'title__secondary'.